layout-and-navigation/lesson-1.md

```diff
@@ -18,7 +18,7 @@ When creating views with Kendo UI, you'll often find yourself using common eleme
 
 * **b**. Add a `data-title="Books"` attribute to the first view (`<div data-role="view">`).
 
-* **c**. Remove the full header div (`<div data-role="header">`) from the first view and copy the NavBar into the div with `data-role="header"`. Your layout should now look like this:
+* **c**. Cut the whole header div (`<div data-role="header">`) from the 'index' view and replace with it the `data-role="header"` in the view you just created. Your layout should now look like this:
```

Comment:
Cut the entire header div (`<div data-role="header">...</div>`) from the `<div data-role="view" id="index" data-title="Books">` block and use it to replace the `<div data-role="header">Header</div>` you added in the previous step. Your layout should now look like this.


Frankly, I had a lot of trouble figuring out what the tutorial wants me to do. o.O This is nightmare.